### PR TITLE
fix(Accordion): fix incorrect props for accordion

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [X] Bug fix 
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to [Typescript typings](./types/lib).
  - [ ] I have updated the typings accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by illiteratewriter** [comment]
fixes #2504

File: stories/examples/AccordionFlush.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import { Accordion, AccordionBody, AccordionHeader, AccordionItem } from 'reactstrap';
 
 const Example = (props) => {
-  const [open, setOpen] = useState();
+  const [open, setOpen] = useState("");
```
Comment (davidacevedo) [inline stories/examples/AccordionFlush.js:5]:
```suggestion
  const [open, setOpen] = useState('');
```
Usually all JS code is in single quotes. I should setup some eslint rules for the project

**Review by davidacevedo — APPROVED** [review]
Looks good, just small nit

File: stories/examples/AccordionFlush.js (hunk 1 above)
Comment (illiteratewriter) [reply stories/examples/AccordionFlush.js:5]: There was an `eslintrc.js` [file](https://github.com/illiteratewriter/reactstrap/blob/25690dcbb1b3a5109b8aac7a2a55223fe38d18b6/.eslintrc.js) which was deleted. Maybe we can get that back. Any idea why it was removed?

**Pull request merged**